spec/09-lexical-structure.md

```diff
@@ -269,7 +269,7 @@ Unless stated otherwise ([functions](13-functions.md#function-definitions),
 
 Names beginning with two underscores (__) are reserved by the PHP language and should not be defined by the user code.
 
-The following names cannot be used as the names of classes, interfaces, or traits: `bool`, `FALSE`, `float`, `int`, `NULL`, `string`, and `TRUE`.
+The following names cannot be used as the names of classes, interfaces, or traits: `bool`, `FALSE`, `float`, `int`, `NULL`, `string`, `TRUE` and `void`.
```

Comment:
@GrahamCampbell Fixed it: https://github.com/php/php-langspec/commit/1935a4b3db0ae797b549870592c60fac6fb74e29